### DISCUSSION TITLE
Compilation errors in TCA module swift-perception using latest Xcode 18.1 beta 2

### DISCUSSION THREAD

**Discussion by njc-esteam** [pointfreeco/swift-composable-architecture#3398] [Q&A]
I tried to build my app using the latest Xcode (18.1 Beta 2 released on 17 Sept) and, without any other changes to the application, Beta 1 completes successfully and Beta 2 throws errors in the Perception module related to Charts dependency failing the compilation.
I have reverted to Beta 1 for now to continue development.
 Any suggestions?

**Comment by mbrandonw** [comment]
Hi @njc-esteam, unfortunately this just seems like an Xcode 16.1 beta bug. I'm sure it is affecting anyone importing Charts. Have you filed a feedback with Apple?
